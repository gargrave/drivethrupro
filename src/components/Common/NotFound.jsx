import React from 'react'
import { Link } from 'react-router-dom'

const NotFound = () => {
  return (
    <div>
      <h2>ONOS! This is the dreaded 404 page!</h2>
      <h4>You are probably looking for one of these pages:</h4>
      <h5>
        <Link to="/orders">View Open Orders</Link>
      </h5>
      <h5>
        <Link to="/orders/new">Create a New Order</Link>
      </h5>
    </div>
  )
}

export default NotFound
