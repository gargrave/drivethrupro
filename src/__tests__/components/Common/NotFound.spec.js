import React from 'react'
import { shallow } from 'enzyme'

import NotFound from '../../../components/Common/NotFound'

test('NotFound renders correctly', () => {
  const component = shallow(<NotFound />)
  expect(component).toMatchSnapshot()
})
