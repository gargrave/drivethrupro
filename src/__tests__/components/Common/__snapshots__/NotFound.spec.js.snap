// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound renders correctly 1`] = `
ShallowWrapper {
  "complexSelector": ComplexSelector {
    "buildPredicate": [Function],
    "childrenOfNode": [Function],
    "findWhereUnwrapped": [Function],
  },
  "length": 1,
  "node": <div>
    <h2>
        ONOS! This is the dreaded 404 page!
    </h2>
    <h4>
        You are probably looking for one of these pages:
    </h4>
    <h5>
        <Link
            replace={false}
            to="/orders"
        >
            View Open Orders
        </Link>
    </h5>
    <h5>
        <Link
            replace={false}
            to="/orders/new"
        >
            Create a New Order
        </Link>
    </h5>
</div>,
  "nodes": Array [
    <div>
      <h2>
            ONOS! This is the dreaded 404 page!
      </h2>
      <h4>
            You are probably looking for one of these pages:
      </h4>
      <h5>
            <Link
                  replace={false}
                  to="/orders"
            >
                  View Open Orders
            </Link>
      </h5>
      <h5>
            <Link
                  replace={false}
                  to="/orders/new"
            >
                  Create a New Order
            </Link>
      </h5>
</div>,
  ],
  "options": Object {},
  "renderer": ReactShallowRenderer {
    "_instance": ShallowComponentWrapper {
      "_calledComponentWillUnmount": false,
      "_compositeType": 2,
      "_context": Object {},
      "_currentElement": <NotFound />,
      "_debugID": 1,
      "_hostContainerInfo": null,
      "_hostParent": null,
      "_instance": StatelessComponent {
        "_reactInternalInstance": [Circular],
        "context": Object {},
        "props": Object {},
        "refs": Object {},
        "state": null,
        "updater": Object {
          "enqueueCallback": [Function],
          "enqueueCallbackInternal": [Function],
          "enqueueElementInternal": [Function],
          "enqueueForceUpdate": [Function],
          "enqueueReplaceState": [Function],
          "enqueueSetState": [Function],
          "isMounted": [Function],
          "validateCallback": [Function],
        },
      },
      "_mountOrder": 1,
      "_pendingCallbacks": null,
      "_pendingElement": null,
      "_pendingForceUpdate": false,
      "_pendingReplaceState": false,
      "_pendingStateQueue": null,
      "_renderedComponent": NoopInternalComponent {
        "_currentElement": <div>
          <h2>
                    ONOS! This is the dreaded 404 page!
          </h2>
          <h4>
                    You are probably looking for one of these pages:
          </h4>
          <h5>
                    <Link
                              replace={false}
                              to="/orders"
                    >
                              View Open Orders
                    </Link>
          </h5>
          <h5>
                    <Link
                              replace={false}
                              to="/orders/new"
                    >
                              Create a New Order
                    </Link>
          </h5>
</div>,
        "_debugID": 2,
        "_renderedOutput": <div>
          <h2>
                    ONOS! This is the dreaded 404 page!
          </h2>
          <h4>
                    You are probably looking for one of these pages:
          </h4>
          <h5>
                    <Link
                              replace={false}
                              to="/orders"
                    >
                              View Open Orders
                    </Link>
          </h5>
          <h5>
                    <Link
                              replace={false}
                              to="/orders/new"
                    >
                              Create a New Order
                    </Link>
          </h5>
</div>,
      },
      "_renderedNodeType": 0,
      "_rootNodeID": 0,
      "_topLevelWrapper": null,
      "_updateBatchNumber": null,
      "_warnedAboutRefsInRender": false,
    },
    "getRenderOutput": [Function],
    "render": [Function],
  },
  "root": [Circular],
  "unrendered": <NotFound />,
}
`;
